pegasus/sites.v3/code.org/views/jumbotron_banner.haml

```diff
@@ -1,3 +1,4 @@
+- specs ||= nil
```

Comment:
It used to be required in jumbotron banner but I don't need it for this page, so made it optional.